Sources/Subscription/SubscriptionCookie/SubscriptionCookieManager.swift

```diff
@@ -21,7 +21,6 @@ import Common
 import os.log
 
 public protocol SubscriptionCookieManaging {
-    init(subscriptionManager: SubscriptionManager, currentCookieStore: @MainActor @escaping () -> HTTPCookieStore?, eventMapping: EventMapping<SubscriptionCookieManagerEvent>)
```

Comment:
the init doesn't belong to the protocol, removing this the entire manager can be easily mocked